src/mc/mcsim.f03

```diff
@@ -129,6 +133,35 @@ pure function list_confinement()
               endif
           endif
 
+        ! set wlc_basepairs to prop if not slide move
+        if (MCTYPE/=13) then 
+            wlc_basepairs_prop = wlc_basepairs
+        endif
+
+         ! sterics check here !
+          if(WLC_P__GJK_STERICS) then
+            if (energyOf(sterics_)%E == 0) then 
+                netSterics = .false.
+            else
+                netSterics = .true.
+            endif
+            call MC_sterics(collisions,netSterics)
+            ! ascribe collision penalty
+            if (netSterics) then 
+                energyOf(sterics_)%dx = collisions 
+            else
+                if (collisions > 0) then 
+                    wlc_ATTEMPTS(MCTYPE) = wlc_ATTEMPTS(MCTYPE) + 1
+                    goto 10 ! skip move, return RP to nan
+                endif
+            endif
+          endif
+
+          ! internucleosome check here !
+            if (WLC_P__INTERNUCLEOSOME_ON) then
+                call mc_internucleosome(left,right)
```

Comment:
It looks like mc_internucleosome no longer has arguments.

Reply:
whoops. thanks for catching that!